canonical/ubuntu.com
# Update meta_image on landing page

## Done

- Update meta_image with new logo on landing page

## QA

- Go to https://ubuntu-com-14093.demos.haus/
- Check that meta image is showing the updated logo: https://assets.ubuntu.com/v1/79356618-ubuntu+core+circular+white.svg

## Issue / Card

Fixes [WD-13502](https://warthogs.atlassian.net/browse/WD-13502)

## Screenshots
meta_image preview
<img width="723" alt="Screenshot 2024-07-23 at 2 39 00 PM" src="https://github.com/user-attachments/assets/f0981205-cb70-41a3-ad20-04e170c1997a">



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13502]: https://warthogs.atlassian.net/browse/WD-13502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
